docs/source/configuration/commandline.rst

```diff
@@ -91,6 +91,8 @@ Html report related
     Skip generation of html report.
 [-noimplied]
     Don't look for implied relationships.
+[-no]
```

Comment:
The option is `--no-orphans`

Reply:
Nice catch! The comment was also wrong ("main graph").